src/monodroid/jni/monodroid-glue.c

```diff
@@ -66,6 +66,8 @@
 #include "xamarin_getifaddrs.h"
 #endif
 
+#define MONO_API_EXPORT __attribute__ ((visibility ("default")))
```

Comment:
Why do you need to add `MONO_API_EXPORT`? Using `MONO_API` should work, as is done for 18 other functions in this file.